source/Server.Extensibility/Server.Extensibility.csproj

```diff
@@ -26,6 +26,8 @@
     <None Remove="Extensions\Infrastructure\Web\Content\**" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Autofac" Version="4.9.4" />
+    <PackageReference Include="Octopus.TinyTypes" Version="0.1.22" />
```

Comment:
Is this needed @shaunmarx ?

Reply:
this is actually repeated?

Reply:
Hmmm nope. It's not. It's a leftover from previous commits